src/test/resources/fixtures/hmh/hmh-insighted/development/caliperEnvelopeForGradeEventWithInstructorGraded.json

```diff
@@ -0,0 +1,90 @@
+{
+  "sensor": "https://www.hmhco.com/Agg-PA",
+  "sendTime": "2019-03-22T13:29:31.905Z",
+  "dataVersion": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "data": [
+    {
+      "id": "urn:uuid:18e9b267-0d58-41ca-bb57-1cc9f86ecc05",
+      "type": "GradeEvent",
+      "actor": {
+        "id": "urn:uuid:194b36cd-56e3-4c20-868f-caeb0b869ff4",
+        "type": "Person"
+      },
+      "action": "Graded",
+      "object": {
+        "id": "urn:uuid:2a85245e-4c45-4b5c-95a8-d67b75379af0",
+        "type": "Attempt",
+        "assignable": {
+          "id": "urn:uuid:43f89aa0-c888-4c92-afed-b6c8784081f4",
+          "type": "Assessment"
+        },
+        "assignee": "urn:uuid:194b36cd-56e3-4c20-868f-caeb0b869ff4"
+      },
+      "eventTime": "2019-03-22T13:29:31.905Z",
+      "generated": {
+        "id": "urn:uuid:89bd9e29-2cfe-43d8-8c47-1a20efab9630",
+        "type": "Score",
+        "extensions": {
+          "recommendedBy": "RENAISSANCE"
+        },
+        "maxScore": 12,
+        "scoreGiven": 6,
+        "attempt": {
```

Comment:
this should be just `"attempt": "urn:uuid:2a85245e-4c45-4b5c-95a8-d67b75379af0"` and not a json block

Reply:
Providing an object for attempt is valid, however it is true that the information in that object is duplicative of the object property, so using attempt with an id with same IRI as used in object makes sense as it will result in a more compact representation of the event